desc/grid.py

```diff
@@ -255,27 +320,48 @@ def inverse_rho_idx(self):
         errorif(
             not hasattr(self, "_inverse_rho_idx"),
             AttributeError,
-            "jit compatible Grid objects do not have inverse indices assigned.",
+            f"{self} does not have inverse indices assigned. "
+            "It is not possible to do this automatically on grids made under JIT.",
         )
         return self._inverse_rho_idx
 
     @property
-    def inverse_theta_idx(self):
-        """ndarray: Indices of unique_theta_idx that recover the theta coordinates."""
+    def inverse_poloidal_idx(self):
+        """ndarray: Indices that recover the unique poloidal coordinates."""
         errorif(
-            not hasattr(self, "_inverse_theta_idx"),
+            not hasattr(self, "_inverse_poloidal_idx"),
             AttributeError,
-            "jit compatible Grid objects do not have inverse indices assigned.",
+            f"{self} does not have inverse indices assigned. "
+            "It is not possible to do this automatically on grids made under JIT.",
         )
-        return self._inverse_theta_idx
+        return self._inverse_poloidal_idx
+
+    @property
+    def inverse_alpha_idx(self):
```

Comment:
What purpose do these functions serve?

Reply:
It's like inverse_rho_idx but for the poloidal coordinate. We usually use them to upsample back to the full grid after an integration so that we can compute things like `data["<|B|>"] * data["sqrt(g)"]` without broadcasting errors.